webapp/portlet/src/main/webapp/vue-apps/people-list/components/PeopleCardFront.vue

```diff
@@ -107,23 +182,25 @@
       </a>
     </div>
 
-    <v-card-text class="peopleCardBody align-center pt-2 pb-1">
+    <v-card-text class="peopleCardBody align-center py-1">
       <a
         :href="url"
         :title="user.fullname"
-        :class="(!user.enabled || user.deleted) && 'text-sub-title' || 'text-color'"
-        class="userFullname text-truncate font-weight-bold d-block">
+        :class="usernameClass"
+        class="userFullname font-weight-bold">
         {{ user.fullname }}
         <span v-if="user.external == 'true' " class="externalFlagClass">
           {{ $t('peopleList.label.external') }}
         </span>
       </a>
-      <v-card-subtitle class="userPositionLabel text-truncate py-0">
+      <v-card-subtitle 
+        class="userPositionLabel text-truncate py-0"
+        :class="isMobile && 'mt-0'">
```

Comment:
Please avoid using JS for Styling, use `mX-Y-` where Y = sm| md | lg | xl + X = a | t | b | e | s (r & l to avoid)
```suggestion
>
```

Reply:
done!